index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title> mourad portfolio </title>
+</head>
+<body>
+    <h1> mourad fam</h1>
+    <section id="About">
+    <h2>About</h2>
+    <p>i love coding with CodeThe Dream</p>
+    <p>im in the intro level Classes</p>
+</section>
+    <H2>Experience</H2>
+    <ul>
+<li>html</li>
+<li>css</li>
+<li>js</li>
+<li>github</li>
+<li>git</li>
+<footer>
+    <p>contact me </p>
+    <ul>
+        <li><a href="Tel:2017830354">phone:201-783-0354</a></li>
```

Comment:
I would also capitalize phone and add space before number.  Example:
Phone:  201-783-0354